### DISCUSSION TITLE
Swagger v2 spec not displaying response body properly

### DISCUSSION THREAD

**Discussion by chrisilander** [scalar/scalar#912] [Q&A] [answered]
Hello. I use [swaggo/swag](https://github.com/swaggo/swag) to generate an API spec for my project (only swagger v2 spec available atm). The generated spec can be displayed properly in redoc, and almost properly in scalar. The only issue I found so far is that the response body "box" is empty unless I manually wrap the schema with:
```
...
"content:" {
  "application/json": {
    "schema": ...
  }
}
```
Using this (the automatically generated spec using `swag`): 
![image](https://github.com/scalar/scalar/assets/157801406/55fbf174-3b08-4ede-9a30-4e77db3b0683)

Will render this:
![image](https://github.com/scalar/scalar/assets/157801406/4195e9c6-3c8b-4f12-b5ae-d7034fa41689)

But wrapping the response body schema as mentioned above, like so:
![image](https://github.com/scalar/scalar/assets/157801406/ef869ca4-0991-4861-b390-42d6a6ff2d79)

Will render it correctly:
![image](https://github.com/scalar/scalar/assets/157801406/c87ab790-0110-43b6-bdc6-f620b0c8bf21)

This does not seem to be an issue using redoc. Does anyone have any input on this?

**Comment by hanspagel** [comment, marked as answer]
Oh, yes! Let’s fix this

**Reply by hanspagel** [reply]
PR #913

Might be a new record: 6 minutes from report to PR. :)

**Reply by chrisilander** [reply]
Amazing! Thank you for the blazingly fast response 🚤💨 
I will be eagerly awaiting the new release.

**Reply by marclave** [reply]
@chrisilander this was released earlier today in latest ✨ 

what a record set by @hanspagel 🏃
